lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2996,7 +2997,7 @@ def errorbar(self, x, y, z, zerr=None, yerr=None, xerr=None, fmt='',
                  barsabove=False, errorevery=1, ecolor=None, elinewidth=None,
                  capsize=None, capthick=None, xlolims=False, xuplims=False,
                  ylolims=False, yuplims=False, zlolims=False, zuplims=False,
-                 arrow_length_ratio=.4, **kwargs):
```

Comment:
note to other reviewers / future self: this kwarg has never been released so safe to directly remove.